api/orchestration_movers.go

```diff
@@ -274,7 +297,8 @@ func (o *datasyncOrchestrator) createDatasyncLocation(ctx context.Context, mover
 		// we need to generate that first, before creating the location
 
 		path := fmt.Sprintf("/spinup/%s/%s/", o.server.org, group)
-		roleName := fmt.Sprintf("SpinupDataSyncAccess-%s-%s", mover, s3Arn.Resource)
+		// TODO: the role name is 64 chars max, so we might exceed it
+		roleName := fmt.Sprintf("Spinup-%s-%s", mover, s3Arn.Resource)
```

Comment:
I think I missed this in the create, but why have a capital `S` here?  i don't think we do that anywhere else.

Reply:
I think I'll just get rid if the `Spinup` since the mover name will already start with `spinup-`